test/e2e/util/kubevirt.go

```diff
@@ -81,80 +76,3 @@ func WaitForKubeVirtCluster(t *testing.T, ctx context.Context, client crclient.C
 
 	t.Logf("KubeVirtCluster is ready in %s", time.Since(start).Round(time.Second))
 }
-
-func createKubeVirtClusterWildcardRoute(t *testing.T, ctx context.Context, client crclient.Client, guestClient crclient.Client, hostedCluster *hyperv1.HostedCluster, baseDomain string) {
-
-	g := NewWithT(t)
-
```

Comment:
we were doing all this DNS/route passthrough logic in the e2e tests. Now that can be stripped out. that's why all this test code is being removed.